app/renderer/components/preferences/payment/bitcoinDashboard.js

```diff
@@ -341,7 +341,7 @@ class BitcoinDashboard extends ImmutableComponent {
                         {/* TODO: refactor button.js */}
                         <Button className={cx({
                           primaryButton: true,
-                          [css(globalStyles.spacing.dialogInsideMargin)]: true
+                          [css(styles.bitcoinAddressButton)]: true
```

Comment:
I would instead set `styles.dialogInsideMargin` here and under [L378](https://github.com/brave/browser-laptop/pull/8382/files#diff-799b11467a9bfc86f7dbbf11fcaee159R378) which is for me more understandable (as long as there is no styles specific to `bitcoinAddressButton`).